chatlogger.lua

```diff
@@ -12,33 +12,39 @@ function OnStartup()
 		sBotName = SetMan.GetString( 21 ) or "PtokaX",
 		sProfiles = "012",		-- No history for commands from users with profiles
 		sLogsPath = "/www/ChatLogs/",
+		sTimeFormat = "[%I:%M:%S %p] ",
```

Comment:
We could change this to `os.date("[%I%M%S %p] ")` like [here] (https://github.com/nishithshah2211/ptokax-scripts/blob/chathistory/chatrooms.lua#L17)